src/main/webapp/app/overview/course-dashboard/course-dashboard.component.html

```diff
@@ -1,9 +1,38 @@
-<div class="row justify-content-center m-4">
+<div class="col-sm col align-items-center">
     <!--
-    Currently work in progress.
-    TODO: A page with two columns.
-        - Left part: Iris Course Chat
-        - Right part: Statistics and competencies.
-    -->
-    <h1 [style]="{ 'text-align': 'center' }">Work in progress</h1>
+        Currently work in progress.
+        TODO: A page with two columns.
+            - Left part: Iris Course Chat
+            - Right part: Statistics and competencies.
+     -->
+    <div class="row justify-content-center w-100">
+        <h1 [style]="{ 'text-align': 'center' }">Work in progress</h1>
+    </div>
+
+    <div class="competency-accordion-container row justify-content-center align-items-center w-100">
+        @if (competencies.length > 0) {
+            <div class="competency-accordion-header">
+                <h2 jhiTranslate="artemisApp.studentAnalyticsDashboard.competencyAccordion.title"></h2>
+            </div>
+            <div class="competency-accordion-body" [ngClass]="this.openedAccordionIndex !== null ? 'competency-accordion-body-open' : 'competency-accordion-body-collapsed'">
+                @for (competency of competencies; track competency.id; let i = $index) {
+                    <jhi-competency-accordion
```

Comment:
Why is there so much whitespace? 
<img width="941" alt="image" src="https://github.com/ls1intum/Artemis/assets/5898705/4887d95e-31f8-47e8-a24a-f756ffbb69da">

Reply:
Seems to be related to the next objective

Reply:
I will look into this.